src/widget/wcoverartlabel.h

```diff
@@ -16,7 +16,8 @@ class CoverInfoRelative;
 class WCoverArtLabel : public QLabel {
     Q_OBJECT
   public:
-    explicit WCoverArtLabel(QWidget* parent = nullptr);
+    explicit WCoverArtLabel(QWidget* parent = nullptr, WCoverArtMenu* pCoverMenu = nullptr);
```

Comment:
Can this become: 
```suggestion
    explicit WCoverArtLabel(QWidget* parent = nullptr, QMenu* pCoverMenu = nullptr);
```
Or is there any dependency to the WCoverArtMenu

Reply:
Ah, it is correct, we need this for the connection

Reply:
The connection only forward signals. Is it possible to connect them directly to the menu and remove all dependency here?

Reply:
Only if we don't use setCoverArt from WCoverArtMenu we can remove the dependency. According to my tests, I didn't notice anything with/without that method.